common/src/main/java/rs/banka4/rafeisen/common/currency/CurrencyCode.java

```diff
@@ -0,0 +1,12 @@
+package rs.banka4.rafeisen.common.currency;
+
+public enum CurrencyCode {
```

Comment:
please also delete the other `CurrencyCode`s (but, thanks for deduplicating this!)